pkg/innerring/innerring.go

```diff
@@ -358,6 +358,12 @@ func New(ctx context.Context, log *logger.Logger, cfg *viper.Viper, errChan chan
 	walletPath := cfg.GetString(walletPathKey)
 	walletPass := cfg.GetString("wallet.password")
 
+	// parse default validators
+	server.predefinedValidators, err = parsePredefinedValidators(cfg)
+	if err != nil {
+		return nil, fmt.Errorf("ir: can't parse predefined validators list: %w", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("can't parse predefined validators list: %w", err)
```
upper context knows that this is ir

Reply:
fixed